python/src/uagents/context.py

```diff
@@ -554,3 +558,11 @@ async def send_raw_exchange_envelope(
             destination=destination,
             endpoint="",
         )
+
+    async def send_wallet_message(
+        self,
+        destination: str,
+        text: str,
+        msg_type: int = 1,
```

Comment:
From the `babble` package:
```
...
    "type": msg_type,  # 1 for text message, 2 for transaction data
...
```
This would be better as an `int, Enum` type, but I think that should go into the `babble` package rather than here.